e2e/pages/events/uploadC21OrderEvent.page.js

```diff
@@ -0,0 +1,37 @@
+const { I } = inject();
+
+module.exports = {
+
+  fields: {
+    orderTitle: '#temporaryC21Order_orderTitle',
+    orderDetails: '#temporaryC21Order_orderDetails',
+    judgeTitleRadioGroup: {
```

Comment:
Can we not re-use the fragment @swalker125 created for judge?

Reply:
good point, I'll trying using that instead - didn't see it as it went into master after I began this story